src/main/EmailService.java

```diff
@@ -0,0 +1,42 @@
+package main;
+
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import static lombok.AccessLevel.PRIVATE;
+
+
+@Service
+@RequiredArgsConstructor
+@FieldDefaults(level = PRIVATE)
+public class EmailService {
+    @Autowired
```

Comment:
Why is this needed if you declare a constructor? And use constructor injection whenever possible